app/src/main-process/menu/build-default-menu.ts

```diff
@@ -278,6 +278,7 @@ export function buildDefaultMenu(
         label: __DARWIN__ ? 'Delete…' : '&Delete…',
         id: 'delete-branch',
         click: emit('delete-branch'),
+        accelerator: 'CmdOrCtrl+Shift+D',
```

Comment:
It would appear that everywhere else we specify `accelerator` before `click` might make sense to do the same for consistency?